src/main/java/org/mozilla/zest/core/v1/ZestAssignRegexDelimiters.java

```diff
@@ -154,6 +154,14 @@ public ZestAssignRegexDelimiters deepCopy() {
 	 */
 	private String getTokenValue(String str) {
 		if (str != null) {
+			// Check if the pattern are set.
+			// They will not be set after a call to copy or a json deserialisation using field reflexion.
+			if (prefixPattern==null) {
```

Comment:
We should avoid duplicate the pattern initialisation. Could you remove the ones being done in the pre/postfix setters?